lib/tool_belt/bugzilla.rb

```diff
@@ -0,0 +1,99 @@
+require 'rest-client'
+require 'json'
+
+class RedHatBugzilla
+
+  HOSTNAME = 'https://bugzilla.redhat.com/jsonrpc.cgi'
+
+  def initialize(user, password)
+    @user = user
+    @password = password
+  end
+
+  def get_bug(id, fields = default_fields)
+    id = [id] if !id.is_a?(Array)
+    request('Bug.get', {
+      :ids => id,
+      :include_fields => fields
+    })
+  end
+
+  def bugs_for_release(id, options = {})
+    status = options.fetch(:status, ["POST"])
+
+    case options[:state]
+    when "approved", "blocker"
+      id = "#{id}+"
+    when "pending"
+      id = "#{id}?"
+    end
+
+    params = {
+      :f1 => "target_milestone",
+      :o1 => "equals",
+      :v1 => "#{id}",
+      :f2 => "flagtypes.name",
+      :o2 => "equals",
+      :v2 => "pm_ack+",
+      :f3 => "flagtypes.name",
+      :o3 => "equals",
+      :v3 => "devel_ack+",
+      :f4 => "flagtypes.name",
+      :o4 => "equals",
+      :v4 => "qa_ack+",
+      :f5 => "flagtypes.name",
+      :o5 => "equals",
+      :v5 => "sat-6.1.z+",
+      :query_format => "advanced",
+      :status => status,
+      :include_fields => default_fields,
+      :product => "Red Hat Satellite 6"
```

Comment:
Can we make this configurable versus hard-coding it here? Given the rest is bugzilla generic?